Directory.Build.props

```diff
@@ -1,4 +1,8 @@
 <Project>
+  <Import
```

Comment:
Maybe this is too paranoid, but can we get some kind of warning that fires if we import this in a CI environment? It could check if `env.TEAMCITY_VERSION` exists for example. Or maybe it warns if this is imported regardless?

Reply:
don't count on TEAMCITY_VERSION....(VSTS builds). But yeah, might be good to condition this with` AND '$(IsOfficialBuild)' != 'true'`

Reply:
Would using '$(CI)' be better? Seems less work than inventing a new property?

Reply:
ok with me

Reply:
Would `$(CI)` exist in Travis AppVeyor and TC? If so that seems fine to me.

Reply:
They exist on AppV and Travis, but don't on TC. We would need to set that one up.

Reply:
If it's only missing on TC we can just add it as an env on the Root Project, that should do the trick for all configs.